server.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const hostname = '0.0.0.0';
+const port = 3000;
+
+// Computes fibonacci:
```

Comment:
Can you create a separate function for this, and add a little more detail about what the function is computing? (It can be good to clarify what the "first" Fibonacci number is since some people start with 0, 1, 1, and other people start with 1, 1, 2.)